clodius/cli/aggregate.py

```diff
@@ -1618,39 +1572,37 @@ def geojson(
     metavar='FILEPATH'
 )
 @click.option(
-    '--output-file',
     '-o',
+    '--output-file',
     default=None,
     help="The default output file name to use. If this isn't"
-    "specified, clodius will replace the current extension"
-    "with .hitile"
+         "specified, clodius will replace the current extension"
+         "with .gjdb"
 )
 @click.option(
-    '--assembly',
-    '-a',
-    help='The genome assembly that this file was created against',
-    type=click.Choice(nc.available_chromsizes()),
-    default='hg19')
-@click.option(
-    '--tile-size',
-    '-t',
-    default=256,
-    help="The number of data points in each tile."
-    "Used to determine the number of zoom levels"
-    "to create.")
-@click.option(
-    '--chromsizes-filename',
-    help="A file containing chromosome sizes and order",
-    default=None)
+    '-m',
+    '--max-per-tile',
+    default=20,
+    type=int
+)
 @click.option(
-    '--starting-resolution',
     '-s',
+    '--tile-size',
     default=256,
-    help="The resolution that the starting data is at (e.g. 1, 10, 20)")
+    help="The number of nucleotides that the highest resolution tiles "
+         "should span. This determines the maximum zoom level"
+)
 @click.option(
-    '--row-infos-filename',
-    help="A file containing the names of the rows in the multivec file",
-    default=None)
-def multivec(filepath, output_file, assembly, tile_size, chromsizes_filename, base_resolution, row_infos_filename):
-    _multivec(filepath, output_file, assembly, tile_size,
-              chromsizes_filename, base_resolution, row_infos_filename)
+    '-z',
+    '--max-zoom',
+    default=19,
+    help="The number of nucleotides that the highest resolution tiles "
+         "should span. This determines the maximum zoom level"
+)
+def geojson(
+    filepath, output_file, max_per_tile, tile_size, max_zoom
+):
+    """Aggregate a geojson file"""
+    _geojson(
```

Comment:
Seems redundant to me, but there might be a larger scheme.

Reply:
You mean the docstring or the `_geojson` method?

Reply:
Just the same thing as above: The methods seem redundant.